### ISSUE TITLE
Share opthon doesn't work on Firefox

### ISSUE THREAD

**Issue by yuuur** [g2glab/hellograph#30]
The share option does not work on firefox. (The chrome one is fine.)

**Comment by ryotayamanaka** [comment]
Thank you for the report.

I have confirmed the issue:
```
Uncaught ReferenceError: ClipboardItem is not defined
https://g2glab.github.io/hellograph/:1123
```

[index.html:1123](https://github.com/g2glab/hellograph/blob/2fe7963a6d8c940c85792f7c20bf3fb4256fd87f/index.html#L1123)

**Comment by shota-matsumoto-lm** [comment]
I fixed this on f664ba150290450481140ec646a6ffe9b5a5d7e5

**Comment by ryotayamanaka** [comment]
Thank you!
